src/PrestaShopBundle/Resources/config/services/core/command_bus.yml

```diff
@@ -11,20 +11,20 @@ services:
         parent: 'prestashop.core.command_bus'
         public: true
 
-    prestashop.core.provider.command_definition_provider:
```

Comment:
It's a BC Break if a service is not available anymore.

Reply:
Indeed we can keep the former service definition (since we kept the service class anyway) And mark it as deprecated, Symfony allows this `deprecated: 'The "%service_id%" service is deprecated. Use "prestashop.core.provider.command_handler_definition_parser" instead'`